apps/backoffice-v2/src/pages/Entity/components/Details/Details.tsx

```diff
@@ -25,9 +32,11 @@ export const Details: FunctionComponent<IDetailsProps> = ({ id, value, hideSepar
         workflowId={workflow?.id}
         id={id}
         valueId={value?.id}
-        documents={workflow?.context?.documents}
+        documents={documents ?? workflow?.context?.documents}
```

Comment:
This change will cause confusion and might result in unexpected outcomes. Let's at least receive documents as props for both cases.